waffle/nn/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 
 class Module:
-  def __init__(self, name:str, path:str, mode:str='list'):
+  def __init__(self, name:str, path:str, mode:str='graph'):
     self.model_name = name; self.onnx_obj = None
     self.linearized_model = read_onnx(path)
     self.onnx_obj = onnxList(self.linearized_model) if mode == 'list' else onnxGraph(self.linearized_model) 
```

Comment:
this line should be 
`self.onnx_obj = onnxGraph(self.linearized_model) if mode == 'graph' else onnxList(self.linearized_model) `